aptos-move/framework/aptos-stdlib/sources/fixed_point64.move

```diff
@@ -29,31 +29,31 @@ module aptos_std::fixed_point64 {
     /// Abort code on calculation result is negative.
     const ENEGATIVE_RESULT: u64 = 0x10006;
 
-    /// Returns x - y. x must be not less than y.
-    public fun sub(x: FixedPoint64, y: FixedPoint64): FixedPoint64 {
-        let x_raw = get_raw_value(x);
+    /// Returns self - y. self must be not less than y.
+    public fun sub(self: FixedPoint64, y: FixedPoint64): FixedPoint64 {
+        let x_raw = get_raw_value(self);
         let y_raw = get_raw_value(y);
         assert!(x_raw >= y_raw, ENEGATIVE_RESULT);
         create_from_raw_value(x_raw - y_raw)
     }
     spec sub {
         pragma opaque;
-        aborts_if x.value < y.value with ENEGATIVE_RESULT;
-        ensures result.value == x.value - y.value;
+        aborts_if self.value < y.value with ENEGATIVE_RESULT;
+        ensures result.value == self.value - y.value;
     }
 
-    /// Returns x + y. The result cannot be greater than MAX_U128.
-    public fun add(x: FixedPoint64, y: FixedPoint64): FixedPoint64 {
-        let x_raw = get_raw_value(x);
+    /// Returns self + y. The result cannot be greater than MAX_U128.
+    public fun add(self: FixedPoint64, y: FixedPoint64): FixedPoint64 {
```

Comment:
the form `z = add(x, y)` is not a convention any longer? are you going with `z = x.add(y)`?

Reply:
I am not sure about this. If `add(x,y)` is a convention most people use, I can revert this change. also cc @movekevin @lightmark @wrwg

Reply:
Also, are there any other conventions in `fixed_point32` or `fixed_point64` such as `greater_or_equal`?

Reply:
I will keep the change now because developers can still use `add(x,y)`